client/app/components/packages/filed-eas/edit.hbs

```diff
@@ -0,0 +1,25 @@
+<div class="grid-x grid-margin-x">
+  <div class="cell large-8">
+
+    <section class="form-section">
+      <h1 class="header-large">
+        Environmental Assessment Statement (EAS)
```

Comment:
This is completely my bad but can you add "Filed" in front of line 6 here? And also help me add "Draft" to the draft edit component? Sorry and thank you!!